SRS/SRS_OC_BODBAW/srs_oc_bodbaw_netcdf_creation.py

```diff
@@ -0,0 +1,856 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" Script to convert Pigment, Absorption or TSS file into Netcdf. CSV and plots
+are also generated"""
+
+from datetime import datetime
+from generate_netcdf_att import generate_netcdf_att, get_imos_parameter_info
+from netCDF4 import Dataset, date2num, stringtochar
+import argparse
+import numpy as np
+import numpy.ma as ma
+import os
+import pandas as pd
+import pyexcel
```

Comment:
because pandas reads a spreadsheet more as a whole. It does it well, but I found pyexcel much easier to use as I need to read a file more line by line, section by section.